StripeFinancialConnections/StripeFinancialConnections/Source/Common/FlowRouter.swift

```diff
@@ -89,11 +89,6 @@ class FlowRouter {
     }
 
     private var shouldUseNativeInstantDebits: Bool {
-        // Show web instant debits flow while UITesting for now.
-        guard ProcessInfo.processInfo.environment["UITesting"] == nil else {
```

Comment:
it seems like you addressed this (I see changed in `PaymentSheetUITest`), but noting regardless that one thing to be careful is that we do have two tests written for instant debits in the payment code so have to be always be careful that they are not accidentally affected

Reply:
Yep! The `PaymentSheetUITests` now use the native instant debits flow, and there are still UI tests for the web instant debits flow in `FinancialConnectionsUITests`